api/manifest.go

```diff
@@ -41,57 +40,96 @@ func apiManifestHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func getManifest(ctx context.Context, manifestAPI manifest.API, sha string, paths []string) (string, []byte, error) {
-	if sha == "" {
-		sha = "latest"
-	}
-	fetchedSHA := sha
+	log := shared.GetLogger(ctx)
+	mc := shared.NewMemcacheReadWritable(ctx, time.Hour*48)
+	// Shorter expiry for latest SHA, to keep it current.
+	latestMC := shared.NewMemcacheReadWritable(ctx, time.Minute*5)
```

Comment:
This can be moved to line 85. see [my comment](https://github.com/web-platform-tests/wpt.fyi/pull/2207/commits/5843e88e16d7a5d2c770257fd0c5b6eaf1140ec6#r508210968)